src/node.cc

```diff
@@ -3688,6 +3695,10 @@ static void ParseArgs(int* argc,
       trace_deprecation = true;
     } else if (strcmp(arg, "--trace-sync-io") == 0) {
       trace_sync_io = true;
+    } else if (strcmp(arg, "--enable-tracing") == 0) {
+      trace_enabled = true;
+    } else if (strncmp(arg, "--enabled-categories=", 21) == 0) {
```

Comment:
Not sure about requiring the `=` in `'--enabled-categories='` seems inconsistent with similar flags like `--require`